templates/default/haproxy.service.erb

```diff
@@ -3,10 +3,10 @@ Description=<%= @name %>
 After=syslog.target network.target
 
 [Service]
-Environment="CONFIG=/etc/haproxy/haproxy.cfg" "PIDFILE=/run/haproxy.pid"
-ExecStartPre=/usr/sbin/haproxy -f $CONFIG -c -q
+Environment="CONFIG=<%= @options['conf_file'] -%>" "PIDFILE=<%= @options['pid_file'] -%>"
+ExecStartPre=<%= @options['bin_prefix'] -%>/sbin/haproxy -f $CONFIG -c -q
 ExecStart=<%= @command %>
-ExecReload=/usr/sbin/haproxy -f $CONFIG -c -q
+ExecReload=<%= @options['bin_prefix'] -%>/sbin/haproxy -f $CONFIG -c -q
```

Comment:
We probably want `bin_prefix/haproxy` so people can throw their haproxy binary in /opt/bin or /usr/local/bin right?

Reply:
In case I understand you correctly, building haproxy from sources creates a dir structure like this:

```
<bin_prefix>
├── doc
│   └── haproxy
│       ├── 51Degrees-device-detection.txt
│       ├── architecture.txt
│       ├── close-options.txt
│       ├── configuration.txt
│       ├── cookie-options.txt
│       ├── DeviceAtlas-device-detection.txt
│       ├── intro.txt
│       ├── linux-syn-cookies.txt
│       ├── lua.txt
│       ├── management.txt
│       ├── netscaler-client-ip-insertion-protocol.txt
│       ├── network-namespaces.txt
│       ├── proxy-protocol.txt
│       ├── SPOE.txt
│       └── WURFL-device-detection.txt
├── sbin
│   ├── haproxy
│   └── haproxy-systemd-wrapper
└── share
    └── man
        └── man1
            └── haproxy.1
```

So the haproxy binary will always be under `<bin_prefix>/sbin/haproxy`, not `<bin_prefix>/haproxy`